gamestonk_terminal/jupyter/dashboards/vsurf.ipynb

```diff
@@ -2,52 +2,54 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 9,
```

Comment:
should these be changed?

Reply:
I think this is metadata of some sort. I have been leaving the stuff the linter doesnt touch alone to avoid breaking things.